website/docs/r/is_backup_policy.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : ibm_is_backup_policy"
+description: |-
+  Manages BackupPolicy.
+---
+
+# ibm_is_backup_policy
+
+Provides a resource for BackupPolicy. This allows BackupPolicy to be created, updated and deleted. For more information, about backup policy in your IBM Cloud VPC, see [Backup policy](https://cloud.ibm.com/docs/vpc?topic=vpc-creating-backup-policy).
+
+## Example Usage
+
+```hcl
```

Comment:
This documentation, needs work for having a correct referential working example.

Reply:
yes, once the full development is done i will revisit the documentation and will inform you to verify